src/login/logind-dbus.c

```diff
@@ -2945,6 +2946,16 @@ int manager_start_scope(
                         return r;
         }
 
+        if (!isempty(requires)) {
+                r = sd_bus_message_append(m, "(sv)", "After", "as", 1, requires);
+                if (r < 0)
+                        return r;
+
+                r = sd_bus_message_append(m, "(sv)", "Requires", "as", 1, requires);
+                if (r < 0)
+                        return r;
+        }
```

Comment:
> (Also, I wonder if this might break things for people who mask user@.service... Maybe this should be Wants= and not Requires=?)

Deps to masked units are ignored, aren't they?